syncmaster/worker/handlers/file/base.py

```diff
@@ -59,14 +60,74 @@ def read(self) -> DataFrame:
         return df
 
     def write(self, df: DataFrame) -> None:
+        tmp_path = os.path.join(self.transfer_dto.directory_path, ".tmp", str(self.run_dto.id))
```

Comment:
```suggestion
        tmp_path = self.transfer_dto.directory_path / ".tmp" / str(self.run_dto.id)
```
target_path should already accept `os.PathLike` objects like `pathlib.Path`

Reply:
Also tmp_path should be removed after writing was finished, regardless of status

Reply:
Used `os.path.join` because `FileTransferDTO` already has a field `directory_path: str`. Do we need to convert this field to a `Path` instead?

Reply:
Sorry, my bad. Let's keep it string for now